pkg/sql/crdb_internal.go

```diff
@@ -1385,10 +1386,15 @@ CREATE TABLE crdb_internal.create_type_statements (
 			switch typeDesc.Kind {
 			case sqlbase.TypeDescriptor_ENUM:
 				var enumLabels []string
+				enumLabelsDatum := tree.NewDArray(types.String)
 				for i := range typeDesc.EnumMembers {
-					enumLabels = append(enumLabels, typeDesc.EnumMembers[i].LogicalRepresentation)
+					rep := typeDesc.EnumMembers[i].LogicalRepresentation
+					enumLabels = append(enumLabels, rep)
+					if err := enumLabelsDatum.Append(tree.NewDString(rep)); err != nil {
+						return err
+					}
 				}
-				name, err := tree.NewUnresolvedObjectName(3, [3]string{typeDesc.GetName(), sc, db.GetName()}, 0)
```

Comment:
Why can the db go away?

Reply:
I pulled in some code from here https://github.com/cockroachdb/cockroach/pull/49808 that is useful in both places. Cross database type references are disallowed, so we aren't going to format the type with its database qualification in these create statements.